azurerm/internal/services/apimanagement/resource_arm_api_management.go

```diff
@@ -88,8 +88,11 @@ func resourceArmApiManagementService() *schema.Resource {
 						"type": {
 							Type:     schema.TypeString,
 							Required: true,
+							Default:  string(apimanagement.None),
```

Comment:
Could we please remove this line? In terraform, a `Required` attribute cannot have a default value.

Reply:
Alright, to correctly set the value to `None` if the entire block is not specified, I presume I'll have to do that in the nil case of the expand method then?

Reply:
I've gone ahead and made that change in 4d1f05b.